release-notes.txt

```diff
@@ -1,3 +1,622 @@
+    * Fix periph_eth link status
+    * Fix reception bug in periph_eth
+    * periph_eth: Use auto-negotation
+* nrf802154:
+    * change internal state to idle after rx
+    * fix set_cca_threshold range
+    * fix unwanted emissions at the end of ACK frame
+    * fix state transition and `off` function
+    * nrf802154: default to netdev_ieee802154_submac
+* cc2538_rf:
+    * fix transmit function
+    * default to netdev_ieee802154_submac
+* at86rf2xx: fix null pointer check
+* at86rf215:
+    + add functions to configure trim & clock output at run-time
+    + implement battery monitor, add power bus
+    + implement MR-FSK
+    * make use of packet queue if available
+    * fix dependency resolution for 2.4 GHz
+* w5100.c: correct the pointer position when w5100's buf is full 
+* srf04: fix overflow on AVR
+* periph/rtt: fix overflow in tick conversion macros
+* mtd_flashpage: fix erasure of multiple sectors
+* mtd_spi_nor: cleanup & bug fixes
+
+Documentation
+-------------
++ README: Add merge-chance badge
++ riotboot: add basic porting guide
++ SECURITY.md: Add initial security policy
+* generate-changelog: add support for point releases
+* add Docker info to doxygen
+* drivers/netdev: Clarify documentation
+* README.md: move tapsetup section to documentation
+* native: Improve Documentation
+* CONTRIBUTING: cleanup and reorganize
+* core/msg_bus: enhace documentation
+* cpu/stm32: rewrite clock configuration documentation
+* doc: advertise forum.riot-os.org
+* doc: Give examples of what DEVELHELP does
+* gnrc/rpl: document implementation status
+* ieee802154/radio: indicate that FCS is ignored when reading buffer
+* uhcp: improve documentation
++ emulators: add emulators documentation in Doxygen
+* update board porting documentation page
+
+Build System / Tooling
+----------------------
++ testbed-support: add iotlab mapping for dwm1001
++ Add support for Github annotations.
++ info.inc.mk: Add info-build-json as JSON-formatted build info
++ makefiles/utils: add util to compare to software versions
++ emulator: add debug feature with qemu emulator
+* Makefile.include: avoid recursive expansion of USEMODULE
+* Github Actions: static-test: use base reference as base branch instead of master
++ makefiles/pseudomodules.inc.mk: netdev_ieee802154_legacy is a pseudomodule
+* gh-actions/release-test: bump ssh-agent to v0.4.1
+* dist/tools/bmp: Fix flashing with pygdbmi 0.10.0
+* drivers/at86rf215: fix dependency resolution for single radio
+* cpu/esp*: fix linker scripts
+* info-objsize: Restore info-objsize target
+* make: run static_tests.sh directly with `static-tests` target 
+* picolibc: model as a feature
+* tools/esptool: add common makefile to define esptool variables
+* dist/tools: Use bash instead of sh where needed
+* tools/emulator: allow use with *term targets + introduce EMULATE=1 instead of emulate target
+* mips: adapt for toolchain update (2020.06-01)
+* use Github actions for static tests instead of Travis
```

Comment:
I think all the points regarding Github actions and static tests above should be summarized under this point.

```suggestion
* use Github actions for static tests instead of Travis
  + Add support for Github annotations.
  * Github Actions: static-test: use base reference as base branch instead of master
```

`* Github Actions: static-test: use base reference as base branch instead of master` is more of a fix to the migration that happened during this release cycle, so not sure if it is worth mentioning here. Note that `* make: run static_tests.sh directly with `static-tests` target ` is not included here, as it is a fix to the static tests script and not the Github Action itself.

Also, it seems odd to me that we don't have another section "Continuous Integration" for topics like this.

Reply:
I also thought about that yesterday while writing the release notes. Maybe we should add it at this point

Reply:
check 33e93cacddb31d3a61c08bbe3b6894cc93c7e85b and a2b2e91ca0026b629d20c647cb7f5908d4b2e36a